data/datavalid.yml

```diff
@@ -456,5 +457,18 @@ schemas:
       - name: stop_and_search_time
       - name: agency
       - name: uid
+  brady_list:
+    columns:
+      - name: brady_uid
+        description: >
+          unique brady uid 
+      - name: uid
+      - name: allegation
+      - name: disposition
+      - name: tracking_number
+      - name: action
+      - name: agency
+      - name: source_agency
+      - name: charging_agency
```

Comment:
To normalize even more, I suggest putting `allegation`, `disposition`, `tracking_number`, `action`, and perhaps `charging_agency` as well in allegation table. And then link to brady list record via `allegation_uid`.

Reply:
If you decide to follow that route then the best place to extract/merge rows is probably the fuse step.